LabController/setup.py

```diff
@@ -1,16 +1,21 @@
 from setuptools import setup, find_packages
-import commands
 from glob import glob
 
+try:
```

Comment:
Martin:  Further down in this file there exists a 'classifier' section which defines the following which perhaps you'd like to change?
        'Programming Language :: Python :: 2.7',
I was looking at this file today and thought of you.